add density leq continuum (#1016)
pi-base/data: properties/P000209.md

---
uid: P000209
name: Density $\leq\mathfrak c$
---

> The cardinality of a dense subset is $\leq \mathfrak c=|\mathbb R|=|2^\omega|$.

Reviewer: ```suggestion
There exists a dense subset with cardinality $\leq \mathfrak c=|\mathbb R|=|2^\omega|$.
```

Otherwise, it seems to be saying every dense subset has cardinality bounded by continuum, which is too strong.

> There exists a dense subset with cardinality $\leq \mathfrak c=|\mathbb R|=|2^\omega|$.

Reviewer 2: ```suggestion
There exists a dense subset with cardinality $\leq \mathfrak c=2^{\aleph_0}=|\mathbb R|$.
```
